subscription-descriptor-parser/src/test/java/uk/gov/justice/maven/generator/io/files/parser/SubscriptionDescriptorFileValidatorTest.java

```diff
@@ -0,0 +1,43 @@
+package uk.gov.justice.maven.generator.io.files.parser;
+
+import static java.nio.file.Paths.get;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.instanceOf;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.fail;
+
+import java.io.IOException;
+
+import org.everit.json.schema.ValidationException;
+import org.junit.Test;
+
+public class SubscriptionDescriptorFileValidatorTest {
+
+    @Test
+    public void shouldPassOnCorrectSubscriptionYaml() throws IOException {
```

Comment:
Maybe be better to say should not throw exception for valid subscription Yaml

Reply:
fixed